app/src/androidTest/java/EspressoBaseTest.kt

```diff
@@ -0,0 +1,72 @@
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.IdRes
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import com.bijoysingh.quicknote.R
+import com.facebook.litho.testing.espresso.ComponentHostMatchers
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+import org.hamcrest.core.AllOf.allOf
+
+object EspressoBaseTest {
```

Comment:
I think, these functions don't need to be in object. It can be just functions without container.